src/main/scala/mesosphere/marathon/core/launchqueue/ReviveOffersConfig.scala

```diff
@@ -21,4 +21,19 @@ trait ReviveOffersConfig extends ScallopConf {
     descrNo = "(Default) Offers will be continually declined for declineOfferDuration.",
     prefix = "disable_"
   )
+
+  lazy val useOfferConstraints = toggle(
+    "use_offer_constraints",
+    default = Some(false),
+    noshort = true,
+    descrYes = "Specify offer constraints to Mesos to avoid receiving some of unneeded offers (experimental).",
```

Comment:
```suggestion
    descrYes = "Send placement constraints to Mesos to reduce the number of offers Marathon needs to decline (experimental).",
```

Reply:
@timcharper Thanks! Makes sense, although I'd strongly prefer not to use the word "placement" in the phrase "Send placement constraints to Mesos" to avoid misleading readers. What we send to Mesos in many cases bears only a slight resemblance to the placement constraints of the app.

Tried to use all the other parts of your suggestion, see https://github.com/mesosphere/marathon/pull/7229/commits/b3ce6d8165da6920fc543c4ea4534e6873576464